thirdParty/alpaka/include/alpaka/mem/buf/cuda/Copy.hpp

```diff
@@ -575,18 +575,17 @@ namespace alpaka
                         auto const & srcNativePtr(task.m_srcMemNative);
 
                         // Fill CUDA parameter structure.
-                        cudaMemcpy3DParms cudaMemCpy3DParms = {};
-                        //cudaMemCpy3DParms.srcArray;     // Either srcArray or srcPtr.
-                        //cudaMemCpy3DParms.srcPos;       // Optional. Offset in bytes.
-
+                        cudaMemcpy3DParms cudaMemCpy3DParms;
+                        cudaMemCpy3DParms.srcArray = nullptr;  // Either srcArray or srcPtr.
+                        cudaMemCpy3DParms.srcPos = make_cudaPos(0, 0, 0);  // Optional. Offset in bytes.
                         cudaMemCpy3DParms.srcPtr =
                             make_cudaPitchedPtr(
                                 const_cast<void *>(srcNativePtr),
                                 static_cast<std::size_t>(srcPitchBytesX),
                                 static_cast<std::size_t>(srcWidth),
                                 static_cast<std::size_t>(srcPitchBytesY/srcPitchBytesX));
-                        //cudaMemCpy3DParms.dstArray;     // Either dstArray or dstPtr.
-                        //cudaMemCpy3DParms.dstPos;       // Optional. Offset in bytes.
+                        cudaMemCpy3DParms.dstArray = nullptr;  // Either dstArray or dstPtr.
+                        cudaMemCpy3DParms.dstPos = make_cudaPos(0, 0, 0);  // Optional. Offset in bytes.
```

Comment:
it's not about how C-structs are set but in what call they are put later on... this member is just not relevant for us since we don't use offsets.